src/spokes/interfaces/IBalanceSheet.sol

```diff
@@ -13,11 +13,18 @@ import {ISpoke} from "src/spokes/interfaces/ISpoke.sol";
 import {IPoolEscrow} from "src/spokes/interfaces/IEscrow.sol";
 import {IPoolEscrowProvider} from "src/spokes/interfaces/factories/IPoolEscrowFactory.sol";
 
-struct QueueAmount {
-    // Issuances of shares / deposits of assets
-    uint128 increase;
-    // Revocations of shares / withdraws of assets
-    uint128 decrease;
+struct ShareQueueAmount {
+    // Net queued shares
+    uint128 delta;
```

Comment:
Storing this as delta now to be able to pack the `queuedAssetCounter` into the same storage slot.